### PR TITLE
add gr-sdrplay3 ebuild

### PR DESCRIPTION
**Pull request by rodan**
added ebuild for an out-of-tree GNU Radio module for SDRplay RSP devices (SDRplay API V3.x)

https://github.com/fventuri/gr-sdrplay3

tested with a RSPdx device and gnuradio-3.10.6.0

### CODE REVIEW

File: net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit cmake python-single-r1
+
+DESCRIPTION="GNU Radio OOT module for SDRplay RSP devices - SRDPlay API v3"
+HOMEPAGE="https://github.com/fventuri/gr-sdrplay3"
+
+if [[ ${PV} == 9999* ]]; then
```
Comment (ZeroChaos-) [inline net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: please fix the whitespace issues in this block

File: net-wireless/gr-sdrplay3/gr-sdrplay3-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit cmake python-single-r1
+
+DESCRIPTION="GNU Radio OOT module for SDRplay RSP devices - SRDPlay API v3"
+HOMEPAGE="https://github.com/fventuri/gr-sdrplay3"
+
+if [[ ${PV} == 9999* ]]; then
```
Comment (ZeroChaos-) [inline net-wireless/gr-sdrplay3/gr-sdrplay3-9999.ebuild:12]: fix whitespace please

File: net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild (hunk 1 above)
Comment (ZeroChaos-) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: this if block has uneven indent

Comment (rodan) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: what are you smoking? I want some too.
all indents are 1 tab long.

Comment (rodan) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: nevermind, I'm blind

Comment (ZeroChaos-) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: Screenshot_2023-06-08_16-25-35

Comment (ZeroChaos-) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: that's even on your screen?

Comment (rodan) [reply net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild:12]: fixed, thanks.

**Comment by ZeroChaos-** [comment]
thanks!

**Pull request merged**